### ISSUE TITLE
No way to edit annotations on a class whose values are defined as enums

### ISSUE THREAD

**Issue by bdionne** [protegeproject/protege#532]
[details](https://github.com/NCIEVS/nci-protege5/issues/12)

**Comment by bdionne** [comment]
This is a feature request, I'd add a tag if I could

**Comment by matthewhorridge** [comment]
To clarify (I can't quite remember how you define these), are you saying that you have something like:

```
AnnotationPropertyRange(:prop :CustomDataType)
DatatypeDefinition(:CustomDataType DataOneOf("a", "b", "c"))
```
And you would like the editor to display a list containing "a","b" and "c" when adding an annotation assertion using ```:prop```?

**Comment by bdionne** [comment]
exactly, a list or a combobox, something that forces the user to select the right value

**Comment by rmehri** [comment]
Hi, 
Can you please share your use case to valid the fix? 
Thanks,
Reza
